packages/cli/internal/pkg/cli/config/config_client.go

```
@@ -135,3 +135,17 @@ func (c Client) GetUserId() (string, error) {
 	userId := userIdFromEmailAddress(userEmailAddress)
 	return userId, nil
 }
+
+func (c Client) SetFormat(format string) error {
+	configData, _ := c.loadFromFile()
```

Why are we ignoring the error here?